### PR TITLE
New composites

### PR DESCRIPTION
**Pull request by t-nelson**
This is built upon #14 and should be merged afterwards. The top three commits are this PR

#### Problem

stake account create + delegate composite transactions require blind signing

#### Changes

* Quells integration test compiler warnings from new Rust version
* Adds some missing tests
* Adds display support for stake account create and delegate composite transaction

### CODE REVIEW

**Comment by t-nelson** [comment]
@CriesofCarrots here too please :slightly_smiling_face:

**Review by CriesofCarrots — APPROVED** [review]
Lgtm!

File: tests/src/main.rs [hunk 1]
```diff
@@ -136,6 +136,39 @@ fn test_ledger_delegate_stake_with_nonce() -> Result<(), RemoteWalletError> {
...
+            ..stake_state::Lockup::default()
+        },
+        42,
+    );
+    let vote_pubkey = Pubkey::new(&[5u8; 32]);
+    let instruction =
+        stake_instruction::delegate_stake(&stake_account, &authorized.staker, &vote_pubkey);
+    instructions.push(instruction);
+    let message = Message::new(&instructions, Some(&ledger_base_pubkey)).serialize();
+    println!("{:?}", message);
```
[the remark this answers is not in the document]
Comment (t-nelson) [reply tests/src/main.rs:166]: Yes. :man_facepalming:

**Comment by lpascal-ledger** [comment]
Conflict resolved & commits merged in one go in [this PR](https://github.com/LedgerHQ/app-solana/pull/16).

**Pull request closed without merging**